server/startup/migrations/v108.js

```diff
@@ -0,0 +1,45 @@
+RocketChat.Migrations.add({
```

Comment:
this migration can be huge, so I think we need to make it asynchronous [migration 106](https://github.com/RocketChat/Rocket.Chat/blob/develop/server/startup/migrations/v106.js#L12)